coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -36,10 +37,31 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
         """
         Add ignore comment
         """
-        comment_delimiter = Language[
-            language].get_default_version().comment_delimiter
-        ignore_comment = (str(comment_delimiter) + ' Ignore ' + result.origin +
-                          '\n')
+        try:
+            singleline_comment_delimiter = Language[
+                language].get_default_version().comment_delimiter
+            ignore_comment = (str(singleline_comment_delimiter) + ' Ignore ' +
+                              result.origin + '\n')
+        except AttributeError:
+            # singleline comments not supported by language
+            try:
+                multiline_comment_delimiter = Language[
+                    language].get_default_version().multiline_comment_delimiters
+                start_multiline_comment = list(
+                                        multiline_comment_delimiter.keys())[0]
+                end_multiline_comment = list(
+                                        multiline_comment_delimiter.values())[0]
```

Comment:
`start_comment, end_comment = multiline_comment_delimiter.items()[0]` should work I think